docs/ivys.js

```diff
@@ -6,7 +6,7 @@
 function IvyApp($grid) {
     "use strict";
     this.$mainGrid = $('#mainGrid');
-    this.$grid = $grid;
+    //this.$grid = $grid;
```

Comment:
Would there not be problems with initializing the $grid before initializing firebase? I changed the order of the code to fit this, but $grid initialization happens before firebase initialization.